package.json

```diff
@@ -17,12 +17,12 @@
     "icons": "webpack --config ./src/icons/webpack.config.js --mode=production",
     "lint": "npm run lint:js && npm run lint:style",
     "lint:no-fix": "npm run lint:js:no-fix && npm run lint:style:no-fix",
-    "lint:js": "vue-cli-service lint ./",
-    "lint:js:no-fix": "vue-cli-service lint --no-fix ./",
+    "lint:js": "eslint --ext .js,.jsx,.vue --ignore-path .gitignore --ignore-pattern svg-sprites.js --fix src",
+    "lint:js:no-fix": "eslint --ext .js,.jsx,.vue --ignore-path .gitignore --ignore-pattern svg-sprites.js --no-fix src",
```

Comment:
We need to run eslint over `tests` and `.storybook` folder too. That's why we were executing the lint service with `./`. We can also control what is ignored with .eslintignore files, e.g. the `svg-sprites.js` is already ignored.